### ISSUE TITLE
Step-by-Step Noob Guide?

### ISSUE THREAD

**Issue by drew5494** [bboyairwreck/PieMessage#8]
Hey would be awesome if you can make one of these for us noobs.  
Thanks!

**Comment by benjyyy** [comment]
that would be great!

**Comment by wesleykunze** [comment]
YES PLEASE

**Comment by danomango** [comment]
Yes indeed please. I am so close but am missing some vital steps. I managed to get the app on my phone and to run but its not connecting to my imessage tools.
